command-line/command-line-interface.md

```diff
@@ -134,13 +134,13 @@ We tend to use kebab-case for directory (folder) names and git repository names.
 8. Use `TAB key` to autocomplete the names of directories and files while in the command line faster as it'll autocomplete the string as far as it can before you have to disambiguate. 
     Hit the tab twice to see the list of AUTOCOMPLETE POSSIBILITES (if multiple matches).
```

Comment:
Could you change this all caps phrase to *italic* or  **bold**?

Reply:
Same with the absolute path and relative path headers below... I’d suggest a 

### header 

or 

#### header